client/src/components/APIClient.ts

```diff
@@ -1,6 +1,18 @@
 /* eslint-disable no-undef */
 
-function searchAddress(q) {
+type WithBoroBlockLot = {
+  boro: string,
+  block: string,
+  lot: string,
+};
+
+type Q = {
```

Comment:
I have no idea what the `q` parameters semantically meant (maybe "query"? 🤷‍♂ ) so I just called the type for it `Q`.